daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/ProcessorStateBases.scala

```diff
@@ -574,7 +575,15 @@ abstract class ParseOrUnparseState protected (
     if (!suppress) {
       val rsdw =
         new RuntimeSchemaDefinitionWarning(warnID, ctxt.schemaFileLocation, this, str, args: _*)
-      diagnostics = rsdw :: diagnostics
+      if (tunable.escalateWarningsToErrors) {
+        val sde = new SchemaDefinitionErrorFromWarning(
+          rsdw,
+          warnID
+        )
```

Comment:
Just curious if this was auto-formatted to move everything to a new line.  Seems a little over-zealous in this case to me.

Reply:
Oh no, I just wrote it that way...it wasn't autoformatted to look like that. I can change it to one line tho